go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/secrethub/secrethub-cli
 
-go 1.14
+go 1.16
```

Comment:
Should this version bump impact Goreleaser or other aspects in our release pipeline?

Reply:
Good point. Will revert this change for now.